src/python/director/drcargs.py

```diff
@@ -132,6 +132,29 @@ def addDefaultArgs(self, parser):
                             default=[],
                             help='Run other python startup scripts in addition to startup.py')
 
+    def getRobotType(self):
```

Comment:
these are unused. 

presumably to needed to add them, but what use have you for them?